addons/OPT/tracker/fn_addMarker.sqf

```diff
@@ -0,0 +1,77 @@
+/**
+* Funktion zum erstellen eines Markers
+* 
+* Autor: [GNC]Lord-MDB
+*
+* Argumente:
+* 0: <ARRAY> _startPosition (Start Position des erstellten Marker)
+* 1: <STRING> _prioritaet (realtime|high|normal|low, aktuellsierungs Angabe des Marker durch den Tracking code)
+* 2: <Objekt> _trackingobjekt (Objekt für die Aktuellsierung des Markers)
+* 3: <STRING> _text (Overlay Textanzeige des Markes)
+* 4: <ARRAY> _icon (Texturverweis für Markersymbol)
+* 5: <Number> _winkel (Ausrichtung des Markers im Overlay)
+* 6: <ARRAY> _farbe (Farbe des Markers in RGB Code)
+* 7: <Number> _sichtbarkeit (Sichtbarkeit des Maker auf der Karte: 0-1)
+* 8: <Number> _ebene (Ebene des Maker auf der Karte: 0-1)
+* 9: <Number> _schriftgroeße (_Schriftgroeße des Maker auf der Karte: 0-50)
+*10: <STRING> _schritart (Schritart Overlay Textanzeige des Markes)
+*
+* Rückgabewert:
+* 0: <Number> _markerID (Marker-ID des erstellten Markers)
+*
+* Server Only:
+* Nein
+* 
+* Lokal:
+* Ja
+* 
+* Global:
+* Nein
+* 
+* API:
+* ja
+* 
+* Beispiel Externer Aufruf:
+* _markerID = [[_startPosition],"_prioritaet",_trackingobjekt,"_text",[_icon],_winkel,[_farbe],_sichtbarkeit,_ebene,_schriftgroeße,"_schritart"] call OFUNC(addMarker);
+* Beispiel interner Aufruf:
+* _markerID = [[_startPosition],"_prioritaet",_trackingobjekt,"_text",[_icon],_winkel,[_farbe],_sichtbarkeit,_ebene,_schriftgroeße,"_schritart"] call OFUNC(addMarker);
+*
+*/
+
+#include "macros.hpp"
+
+params
+					
+[
+	["_startPosition",[0,0,0]],
+	["_prioritaet","realtime"],
```

Comment:
Default sollte die niedrigste Priorität sein, damit wir den Benutzer ermutigen so wenig CPU-zeit wie möglich zu fressen.

Reply:
beantwortet dazu mal die Frage im Forum, wenn keine 4 feste Klassen geben soll, muss hier sowieso eine Zahl rein und kein String.

Reply:
Ich würde vorschlagen als default "normal" zu verwenden

Reply:
> beantwortet dazu mal die Frage im Forum, wenn keine 4 feste Klassen geben soll, muss hier sowieso eine Zahl rein und kein String.

In beiden Fällen würde hier immer eine Konstante als Defaultwert stehen. Die Konstante hat dann entweder ein String oder eine Zahl als Wert.

Reply:
ich mache da Erstmal 2 Stufen Realtime und Normal, default wäre dann Normal mit 40ms zeit.